x-pack/plugins/infra/server/routes/infra/lib/helpers/query.ts

```diff
@@ -86,6 +85,27 @@ export const runQuery = <T>(
     );
 };
 
+export const systemMetricsFilter = {
+  must: [
+    {
+      bool: {
+        should: [
+          {
+            term: {
+              'event.module': 'system',
+            },
+          },
+          {
+            term: {
+              'metricset.module': 'system',
```

Comment:
Same here: This is added because of the older metricbeat versions (I checked the test data and added it after, details https://github.com/elastic/kibana/issues/176403#issuecomment-1954232722)

Reply:
Added a comment in the code on this line, closing this